README.md

```diff
@@ -179,7 +185,12 @@ for alpha testing, but may build on it to implement the dialog.
 
 #### Prerequisites
 
-TODO
+1. Unix-Like Operating System, Debian and Ubuntu are most used, https://www.debian.org/index.pt.html and https://ubuntu.com/;
+2. NGINX as proxy server, https://docs.nginx.com/;
+3. NaviServer as webserver, https://naviserver.sourceforge.io/;
+4. PostgreSQL as database manager system and PSQL as SQL language script. https://www.postgresql.org/docs/ and https://www.postgresql.org/docs/9.2/app-psql.html;
+5. Python3 and TCL-TK as programing languages, https://www.python.org/doc/ and http://tcl.tk/man/tcl8.6/contents.htm;
```

Comment:
Do all of these need to be setup in advance of setting up the project?  The goal for everything under Setup is to help new people get started on the project, so it helps to break it down for them, or direct them to where they can get a list of steps to get things set up.  You might want to check out the [Prerequisites section](https://github.com/F4IF/ctree-demo#prerequisites) from the ctree-demo project.